library/src/main/java/nl/surfnet/nonweb/sso/SSOServiceActivity.java

```diff
@@ -83,6 +87,7 @@ public static void authorize(@NonNull final Context context, //
         _callback = callback;
         _consumerId = consumerId;
         _endpoint = endpoint;
+        _scheme = scheme;
```

Comment:
If scheme would be empty, it will not break, so an additional check isn't necessary